src/server/origin/summary/controller.js

```diff
@@ -13,6 +14,15 @@ export const originSummaryGetController = {
     const origin = Origin.fromState(req.yar.get('origin'))
     const { isValid, result } = origin.validate()
 
+    const pages = []
+
+    /** @type any */
```

Comment:
Without the squigglies, jsdoc ignores this - fallen foul of this many times!

Also, could the type here be Page?  So 

```/** @type Page */```